tests/parsing/GlobbingTest.py

```diff
@@ -331,7 +333,8 @@ def test_no_dirname_recursive(self):
         results = sorted([os.path.normcase(os.path.join(os.curdir, g))
                           for g in glob(pattern)])
         file_list = sorted([os.path.normcase(f) for f in file_list])
-        self.assertEqual([i for i in results if "__pycache__" not in i],
+        self.assertEqual([i for i in results if re.search(r"(__pycache__|.pyc)",
```

Comment:
you enclose the regex inside braces, so you declare a new regex group. You don't need it :)